login-fire-social.html

```diff
@@ -0,0 +1,158 @@
+<!--
+@license
+Copyright (c) 2016 Convoo All Rights Reserved.
+Use of this source code is governed by a MIT license that can be found in the
+LICENSE file or at https://github.com/convoo/login-fire/blob/master/LICENSE.
+-->
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="./login-fire-common-behavior.html">
+<link rel="import" href="../iron-flex-layout/iron-flex-layout.html">
+<link rel="import" href="./login-fire-button.html">
+
+<!--
+An element that allows simple configuration of multiple social provider login with firebase. This web component is split in smaller elements that can be used for email or social only authentication.
+
+Example:
+```
+<firebase-app
+  name="myApp"
+  api-key="API_KEY"
+  auth-domain="AUTH_DOMAIN"
+  database-url="DATABASE_URL"></firebase-app>
+<login-fire-social app-name="myApp" providers="google, facebook, twitter, github, anonymous"></login-fire-social>
+```
+
+@demo demo/login-fire-social.html
+-->
+<dom-module id="login-fire-social">
+  <template>
+    <style>
+      :host {
+        @apply(--layout-vertical);
+        @apply(--layout-center-center);
+      }
+
+      :host([horizontal]) {
+        @apply(--layout-horizontal);
+        @apply(--layout-wrap);
+        @apply(--layout-around-justified);
+        max-width: inherit;
+      }
+
+      login-fire-button {
+        margin-bottom: 15px;
+      }
+
+      :host([horizontal]) login-fire-button {
+        margin-bottom: 0;
+      }
+    </style>
+
+    <template is="dom-repeat" items="{{_providers}}">
+      <login-fire-button
+        app-name="[[appName]]"
+        provider="[[item.id]]"
+        user="{{_user}}"
+        status-known="{{_statusKnown}}"
```

Comment:
status-known is read only and I'm not really sure how that helps with signed-in. Does it mean that if you sign in to one you can sign out from all the other buttons? Doesn't really make much sense to me.
[See docs](https://www.webcomponents.org/element/firebase/polymerfire/firebase-auth) for readOnly mention.